packages/playwright/src/worker/testInfo.ts

```diff
@@ -309,9 +319,7 @@ export class TestInfoImpl implements TestInfo {
     };
     const parentStepList = parentStep ? parentStep.steps : this._steps;
     parentStepList.push(step);
-    const hasLocation = data.location && !data.location.file.includes('@playwright');
     // Sanitize location that comes from user land, it might have extra properties.
```

Comment:
This comment is no longer true. That said, perhaps it was important to remove extra properties?